tests/flows/check_upgrade_access.go

```diff
@@ -0,0 +1,89 @@
+package flows
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+	. "github.com/onsi/gomega"
+)
+
+func CheckUpgradeAccess(network interfaces.Network) {
+	subnetInfo := network.GetPrimaryNetworkInfo()
+	_, fundedKey := network.GetFundedAccountInfo()
+
+	//
+	// Deploy ExampleMessenger to Subnets A and B
+	//
+	ctx := context.Background()
+	teleporterAddress := network.GetTeleporterContractAddress()
```

Comment:
Since we're transferring ownership in this test, I think we'll need to either deploy a new Teleporter contract (and not update the network info) or update the funded account info at the end of this test. My preference would be the first choice.

Reply:
I think this is the same question I posed [here](https://github.com/ava-labs/teleporter/pull/258/files#r1468982194)

Reply:
Right, right, ownership is through the example messenger, not through Teleporter. Thanks.